CONTRIBUTING.md

```diff
@@ -1,5 +1,21 @@
-# Contributing Guidelines
+# Contributing
 
-Add specific contributing guidelines to this file as needed for the individual project.
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
 
-Questions about contributing? [Join our Discord community](https://discord.gg/Byjr6rdBUZ)
```

Comment:
I think we should keep this line. It encourages our users to join our Discord community, and we want to grow our Discord community more and more.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I was planning to put few links, including this one, in readme.md file. But yes, we should keep it here too. Will do the needful.